frontend/static/js/pages/play.js

```diff
@@ -209,11 +230,17 @@ let app = new Vue({
             if (this.path.length == 0 || this.path[this.path.length - 1]["article"] != page) {
                 // Update path
                 let timeElapsed = (Date.now() - this.startTime) / 1000;
-                this.path.push({
-                    "article": page,
-                    "timeReached": timeElapsed,
-                    "loadTime": loadTimeSeconds
-                });
+		
+		    if (this.isPenaltyMode) {
+		        timeElapsed += this.path.length * 20;
+		        loadTimeSeconds -= 20;
+		    }
```

Comment:
This hack is fine for the demo, but for production we need to clean it up. I suggest going with Dan's approach and adding a new field to our schema JSON, maybe `penaltyTime` and we can compute the time on the client by adding `timeElapsed + penaltyTime`.